postgres-protocol/src/message/backend.rs

```diff
@@ -33,6 +36,24 @@ pub const PARAMETER_DESCRIPTION_TAG: u8 = b't';
 pub const ROW_DESCRIPTION_TAG: u8 = b'T';
 pub const READY_FOR_QUERY_TAG: u8 = b'Z';
 
+// replication message tags
+pub const XLOG_DATA_TAG: u8 = b'w';
+pub const PRIMARY_KEEPALIVE_TAG: u8 = b'k';
+
+pub trait Parse: Sized {
```

Comment:
Why is a new trait being introduced just to parse replication messages?

Reply:
This was added to support external plugins using the `DecodingPlugin` trait but I'm happy to remove this. See my other comment on `src/replication/mod.rs` for a counter-proposal that gets rid of that